libyang/log.py

```diff
@@ -26,6 +29,16 @@ def get_libyang_level(py_level):
     return None
 
 
+@contextmanager
+def temp_log_options(opt: int = 0):
+    _ffi = cffi.FFI()
+    opts = _ffi.new("uint32_t *", opt)
```

Comment:
You already have an ffi object available and imported, use it instead of creating a new one.

```python
opts = ffi.new("uint32_t *", opt)
```